examples/changed-signal/init.lua

```diff
@@ -40,9 +42,9 @@ return function()
 	local TextReverser = Roact.Component:extend("TextReverser")
 
 	function TextReverser:init()
-		self.state = {
```

Comment:
we may be changing our guidance on this in the future, so leave this one for now.